src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();

```

Comment:
Don't use static methods or static variables in your solution unless it's necessary. Here, having 'RANDOM' as a static variable is not required and could lead to issues if 'FigureSupplier' is used in a multi-threaded context.